Dockerfile

```diff
@@ -1,4 +1,68 @@
-FROM alpine:latest
+FROM debian:12.5-slim AS stage1
+
+# Supermicro SUM
+# Note: If we remove the SUM tool, we can move back to an alpine image. Then also compile bioscfg with CGO_ENABLED=0
+WORKDIR /tmp/sum
+
+## Pre-reqs
+RUN apt-get update
+RUN apt-get install wget -y
+
+## Download
+RUN wget https://www.supermicro.com/Bios/sw_download/698/sum_2.14.0_Linux_x86_64_20240215.tar.gz -O sum.tar.gz
+RUN mkdir -p unzipped
+RUN tar -xvzf sum.tar.gz -C unzipped --strip-components=1
+
+## Install
+RUN cp unzipped/sum /usr/sbin/sum #TODO; smc sum has the same name as the gnu command sum (/usr/bin/sum). So we are overwritting it. Sorry not Sorry.
+RUN chmod +x /usr/sbin/sum
+
+# IPMI Tool
+#
+# cherry-pick'ed 1edb0e27e44196d1ebe449aba0b9be22d376bcb6
+# to fix https://github.com/ipmitool/ipmitool/issues/377
+#
+ARG IPMITOOL_REPO=https://github.com/ipmitool/ipmitool.git
+ARG IPMITOOL_COMMIT=19d78782d795d0cf4ceefe655f616210c9143e62
+ARG IPMITOOL_CHERRY_PICK=1edb0e27e44196d1ebe449aba0b9be22d376bcb6
+ARG IPMITOOL_BUILD_DEPENDENCIES="git curl make autoconf automake libtool libreadline-dev"
```

Comment:
For solr, we just install ipmitool with apt-get, and then copy `enterprise-numbers.txt` from the solr repo to the expected directory. That way, we aren't dependent on the IANA servers at runtime.

Reply:
My understanding we are doing this for a bugfix in ipmitool. Thats why the cherry pick is there. We dont want to up the version, or change the version because we cant trust ipmitool to not change its functionality.

Reply:
the thing about the IANA servers makes sense though. Maybe we can talk about it.

Reply:
Also, if we should do this. Then we much also do this for flipflop. It has the same issue. This should be a separate ticket.